mautic/mautic
# Asset upload JS validations and preview

How to test:
- If uploaded file is bigger that max size or has file extension which isn't whitelisted, it should throw an error before the upload request.
- Uploaded image should update preview. If there is an error, previous preview will return.
